### ISSUE TITLE
Remove the test helper constant NoS3StoreAvailable

### ISSUE THREAD

**Issue by raghavendra-talur** [RamenDR/ramen#1710]
In file internal/controller/ramenconfig.go
https://github.com/RamenDR/ramen/blob/d5a1bfb83dbd5ae1a52a9bfa6e922ffa41517c6e/internal/controller/ramenconfig.go#L50-L51

**Comment by Ankush1oo8** [comment]
Just to remove it

**Comment by Ankush1oo8** [comment]
Just to remove it

**Comment by raghavendra-talur** [comment]
> @raghavendra-talur Just to remove it

@Ankush1oo8 No, the constant is being used in multiple places in the code and those parts need to be refactored before this line is deleted. Needs understanding of basic ramen workflow to make the changes.

**Comment by sureshkrishnan-v** [comment]
Hi, I’d like to work on this issue as a first-time contributor—could you confirm if it’s available and share any guidelines?
